src/utils/components/NodeSelectorModal/components/LabelRow.tsx

```diff
@@ -46,9 +46,11 @@ const LabelRow: FC<LabelRowProps> = ({ label, onChange, onDelete }) => {
 
       <GridItem span={1}>
         <FormGroup label=" " fieldId={`label-${id}-delete-btn`}>
-          <Button variant="plain" onClick={() => onDelete(id)}>
-            <MinusCircleIcon />
-          </Button>
+          <Tooltip content={t('Remove label selector')}>
+            <Button variant="plain" onClick={() => onDelete(id)}>
```

Comment:
You can use `ButtonVariant.plain` for the `variant` prop instead ;) 
(and import ButtonVariant from the same package as Button, of course)